update_runner.py[LINE:24]# INFO     [2018-05-24 13:27:17,694]  
Starting...
updaters.py[LINE:78]# INFO     [2018-05-24 13:27:22,232]  Executing query...
updaters.py[LINE:81]# INFO     [2018-05-24 13:27:24,088]  Executing finished!
update_runner.py[LINE:33]# INFO     [2018-05-24 13:27:24,088]  Starting GTPRGE_GEN dict comparison ...
updaters.py[LINE:153]# INFO     [2018-05-24 13:27:24,097]  0 RGEs were not found in RIO
